### DISCUSSION TITLE
GitHub discussions beta

### DISCUSSION THREAD

**Discussion by mckennapsean** [dwyl/learn-to-send-email-via-google-script-html-no-server#399] [Ideas]
Would we want to try it out?
https://dzone.com/articles/github-discussions-bringing-the-open-source-commun

**Comment by mckennapsean** [comment]
I do not have access to the repo's settings:
https://github.com/dwyl/learn-to-send-email-via-google-script-html-no-server/settings

It would appear in the list like this:
![Screenshot from 2021-02-07 12-12-58](https://user-images.githubusercontent.com/1406149/107156856-f4728700-693d-11eb-9d2a-c520055dd38c.png)

And in the setup you can create discussions that are general, ideas, Q&A, and Show & Tell. Pretty neat!

**Comment by nelsonic** [comment]
Hi @mckennapsean you should have "maintainer" permissions for this repo. 
Discussions enabled:
![github-discussions](https://user-images.githubusercontent.com/194400/107203849-da937b80-69f3-11eb-9ebb-951a82380c48.png)

Thanks again for proactively opening this issue. 🎉

**Comment by mckennapsean** [comment]
Sweet, I do see more options now, thank you! I'll try moving this thread over to a discussion as a "test", and maybe could triage issues or encourage people to file discussions for questions. Not sure if there's much practical difference, apart from not cluttering bugs vs users' requests for help/assistance? :thinking:

**Comment by mckennapsean** [comment]
Converted! The old issue was closed, and on this page we have the new conversation in the "ideas" channel here. Leaving open for now, in case others have thoughts :) 

Here's a page for Q&A:
https://github.com/dwyl/learn-to-send-email-via-google-script-html-no-server/discussions/categories/q-a

And show & tell:
https://github.com/dwyl/learn-to-send-email-via-google-script-html-no-server/discussions/categories/show-and-tell

**Reply by igor98bn** [reply]
Thanks you for the topics
